integrationExamples/gpt/gpt_sizemapping.html

```diff
@@ -0,0 +1,251 @@
+<html>
+
+<head>
+
+<script>
+
+    var rightSlotCode = '/19968336/header-bid-tag-0';
+    var rightSlotSizes = [[300, 250], [300, 600]];
+
+    var topSlotCode = '/19968336/header-bid-tag1';
+    var topSlotSizes = [[728, 90], [970, 90]];
+
+    var dynamicSizes = [[[1024, 768], [970, 250]],[[980, 690], [728, 90]],[[640, 480], [120, 60]],[[540, 480], [120, 60]],[[0,0],[2,2]]];
+
+    
+	var pbjs = pbjs || {};
+    pbjs.que = pbjs.que  || [];
+    var PREBID_TIMEOUT = 1000;
+
+    pbjs.que.push(function(){
+    
+        var adUnit1 = {
+            code: rightSlotCode,
+            dynamicSizes: dynamicSizes,
```

Comment:
How about this format?

{
   "sizes":[
      {
         "browser":[1024,768],
         "slot":[[970,250],[970,230]]
      },
      {
         "browser":[980,690],
         "slot":[[728,90],[640,480]]
      }
   ]
}